pkg/ctxhelper/ctxhelper.go

```diff
@@ -0,0 +1,52 @@
+package ctxhelper
+
+import (
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/julienschmidt/httprouter"
+	"github.com/flynn/flynn/Godeps/_workspace/src/golang.org/x/net/context"
+	log "github.com/flynn/flynn/Godeps/_workspace/src/gopkg.in/inconshreveable/log15.v2"
+)
+
+type ctxKey string
+
+const (
+	ctxKeyComponent ctxKey = "component"
+	ctxKeyReqID            = "req_id"
+	ctxKeyParams           = "params"
+	ctxKeyLogger           = "logger"
```

Comment:
@titanous any reason I shouldn't just change `ctxKey` to an `int` as done in the blog post?